is_polys: true
area_flying_slope: [ 24,480,90,480,90,750,24,750 ]
area_shooting_windmill: [ 90,420,160,420,160,500,90,500 ]
area_lobbing_1: [ 500,260,670,260,670,460,500,340 ]
area_lobbing_2: [ 500,480,680,600,640,650,480,540 ]
area_refilling: [ 40,100,200,100,200,170,40,170 ]


